### PR TITLE
react-parent-child

### PR DESCRIPTION
**Pull request by Jitta-Sri-Vidhya**
react parent child assignment

### CODE REVIEW

File: parent-child/src/components/MuiCard.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        width: '0px'
+    }
+))
+
+export const MuiCard = () => {
+    return (
+       <Paper sx={{
+        width:'550px',
+        height:'auto',
+        margin:'30px 500px',
```
Comment (kamalabhishek) [inline parent-child/src/components/MuiCard.tsx:30]: Remove inline styling

File: parent-child/src/components/MuiCard.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                <MuiButton variantname='contained'>BUY NOW</MuiButton>
+            </Box>
+       </Paper>
+    )
+}
+
+
+//PrimaryText='Payment method' SecondaryText='Visa credit ...8845'
+//                    <MuiIconTypo heading="Payment method" body="Visa credit ...8845"  Icon={<PaymentOutlinedIcon/>}></MuiIconTypo>
+// <Stack direction='row' sx={{ padding:'8px 20px' }}>
```
Comment (kamalabhishek) [inline parent-child/src/components/MuiCard.tsx:149]: Remove the commented code

File: parent-child/src/components/MuiIconTypo.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { MuiIcon } from '../atoms/MuiIcon';
+import { List, ListItem, ListItemAvatar,ListItemText, Avatar} from '@mui/material';
+import { MuiTypography } from '../atoms/MuiTypography';
+import { Box } from '@mui/material';
+interface Props {
+    // PrimaryText: string
+    // SecondaryText: string
+    // Icon: React.ElementType
```
Comment (kamalabhishek) [inline parent-child/src/components/MuiIconTypo.tsx:8]: Remove commented code

File: parent-child/src/components/MuiIconTypo.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    Icon: JSX.Element
+    children?: JSX.Element|JSX.Element[];
+    heading: string
+    body: string
```
Comment (kamalabhishek) [inline parent-child/src/components/MuiIconTypo.tsx:12]: Place the optional props at the end

**Review by kamalabhishek — APPROVED** [review]

File: parent-child/src/components/MuiCard.tsx (hunk 1 above)
Comment (Jitta-Sri-Vidhya) [reply parent-child/src/components/MuiCard.tsx:27]: removed the inline styling and used the styled componet

File: parent-child/src/components/MuiCard.tsx (hunk 2 above)
Comment (Jitta-Sri-Vidhya) [reply parent-child/src/components/MuiCard.tsx]: i have removed the commented code

File: parent-child/src/components/MuiIconTypo.tsx (hunk 3 above)
Comment (Jitta-Sri-Vidhya) [reply parent-child/src/components/MuiIconTypo.tsx]: I have removed the commented code

File: parent-child/src/components/MuiIconTypo.tsx (hunk 4 above)
Comment (Jitta-Sri-Vidhya) [reply parent-child/src/components/MuiIconTypo.tsx:8]: changed the optional props to the end

**Review by kamalabhishek — APPROVED** [review]

**Pull request merged**